plugins/shell/src/process/mod.rs

```diff
@@ -511,7 +511,7 @@ mod tests {
         assert_eq!(String::from_utf8(output.stdout).unwrap(), "");
         assert_eq!(
             String::from_utf8(output.stderr).unwrap(),
-            "cat: test/: Is a directory\n"
+            "cat: test/: Is a directory\n\n"
```

Comment:
does that sound okay? why does it have two new lines?

Reply:
the std one only gives us a single \n so it's a bug on our end (it was already a bug, since it had a \n where it shouldn't, I guess someone just fixed the test instead of the actual bug). We should track this on the tauri repo (v1 and v2).

Reply:
Cool